### ISSUE TITLE
What's New listing

### ISSUE THREAD

**Issue by DeanGross** [MicrosoftDocs/microsoft-365-docs#2486]
[Enter feedback here]
Please add a What's New/Change Log page to  make it easier to keep track of changes to the Sensitive Info Types

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ba098f73-3ad2-9850-d2f4-6667e15a7be2
* Version Independent ID: e171ad34-39a5-82d1-b3ac-716b1ae3dd7e
* Content: [Sensitive information type entity definitions - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/sensitive-information-type-entity-definitions?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/sensitive-information-type-entity-definitions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/sensitive-information-type-entity-definitions.md)
* Service: **o365-seccomp**
* GitHub Login: @chrfox
* Microsoft Alias: **chrfox**

**Comment by msbemba** [comment]
@DeanGross Thank you for your feedback. I have added a section to the document "What is New?" so that we can track the changes or additional going forward. I'm submitted the PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/2533 for approval and once it is approved you will see the document updated.

Thanks
Sri

**Comment by chrfox** [comment]
@DeanGross - This topic is a highly structured reference topic, use it like a dictionary. A What's new section would required double listing or linking and doesn't fit with the topic.  This change was not merged into the topic.
